python/py-moviepy/Portfile

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-moviepy
+version             1.0.3
+revision            0
+categories-append   graphics
+
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+
+description         Video editing with Python
+
+long_description    MoviePy (full documentation) is a Python library \
+                    for video editing: cutting, concatenations, title \
+                    insertions, video compositing (a.k.a. non-linear \
+                    editing), video processing, and creation of \
+                    custom\ effects. See the gallery for some examples \
+                    of use.MoviePy can read and write all the most \
+                    common audio and video formats, including GIF, and \
+                    runs on Windows/Mac/Linux, with Python 2.7+ and 3 \
+                    (or only Python 3.4+ from v.1.0).
+
+homepage            https://zulko.github.io/moviepy/
+
+master_sites        pypi:m/${python.rootname}/
+distname            ${python.rootname}-${version}
```

Comment:
you can remove `master_sites` and `distname`, these are the defaults set by the python PG

Reply:
Thanks, done.